templates/todo/api/js/src/config/index.ts

```diff
@@ -53,7 +52,7 @@ const populateEnvironmentFromKeyVault = async () => {
 
     try {
         logger.info("Populating environment from Azure KeyVault...");
-        const credential = new ChainedTokenCredential(new AzureDeveloperCliCredential(), new DefaultAzureCredential());
+        const credential = new DefaultAzureCredential({});
```

Comment:
See: https://github.com/Azure/azure-sdk-for-js/issues/25089
about using `{}` when creating the DefaultAzureCredential